liquid_node/jobs/__init__.py

```diff
@@ -21,6 +21,10 @@ def set_volumes_paths(substitutions={}):
 
     from ..configuration import config
 
+    def get_str(str):
```

Comment:
Please remove all this in favor of `raw` and `endraw` tags, see comment below.

Also, a wrapper would not be needed here, you could have done `substititutions['getstr'] = str`, or even better, keep the id as `str` instead of `getstr`.